File: src/main/java/net/maple/handlers/misc/CashShopQueryCashRequestHandler.java
package net.maple.handlers.misc;

import client.Client;
import net.maple.handlers.PacketHandler;
import net.maple.packets.CashShopPackets;
import util.packet.PacketReader;

public class CashShopQueryCashRequestHandler extends PacketHandler {

    @Override
    public void handlePacket(PacketReader reader, Client c) {
        CashShopPackets.sendCashData(c);
    }
}

File: src/main/java/net/maple/handlers/misc/CashShopQueryCashRequestHandler.kt
package net.maple.handlers.misc

import client.Client
import net.maple.handlers.PacketHandler
import net.maple.packets.CashShopPackets.sendCashData
import util.packet.PacketReader

class CashShopQueryCashRequestHandler : PacketHandler {

    override fun handlePacket(reader: PacketReader, c: Client) {
        c.sendCashData()
    }
}